content/docs/concepts/multicluster-configuration/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Multicluster Istio
+description: Describes how a service mesh can be configured to include services from more than one cluster.
+weight: 60
+---
+
+Istio is a [service mesh](/docs/concepts/what-is-istio/#what-is-a-service-mesh),
+the fundamental property of which is its ability to monitor and manage
+a network of cooperating microservices under a single administrative domain,
+essentially combining the microservices into a controllable composite application.
+For applications up to a certain size, all of the microservices comprising the
+application can be running on a single orchestration platform (e.g., kubernetes cluster),
+but for many reasons (e.g., scale, redundancy, etc.) most applications will eventually
+need to be distributed and have some of their services running elsewhere.
+
+Istio supports many possible topologies for distributing the services of an
+application beyond a single cluster.
+Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+to access standalone external services or services exposed by another loosely-coupled
+service mesh, a.k.a. *mesh federation*.
+Alternatively, if all the services can be under the same administrative control,
+you can [expand the service mesh](/docs/setup/kubernetes/mesh-expansion/) to include services running
+on VMs or bare metal hosts, or you can combine the services from more than one cluster into a
+single composite service mesh, i.e., a *multicluster mesh*.
+
+## Multicluster service mesh
+
+A multicluster service mesh is defined as a single composite service mesh composed of services
+running in more than one underlying cluster with all services running
+under a single administrative control.
+A service named `foo` in cluster 1 is the same service as `foo` in cluster 2. This is
+unlike a loosely-coupled federated service mesh where two clusters may have different
+definitions of the same service, which need to be reconciled when integrating them.
+
+A multicluster mesh has the advantage that all the services look the same to clients,
+regardless of where the workloads are actually running. It's transparent
+to the application whether it's deployed in a single or multicluster mesh.
+To achieve this, all of the services need to be managed by a single logical
+control plane, although not necessarily a single physical Istio control plane.
+There are two possible deployment approaches:
+
+1. A single Istio control plane with access to and configuration for all the services in the mesh.
+
+1. Multiple synchronized Istio control planes with replicated service and routing configurations. 
+
+Even within these two topologies, there is more than one way to configure things.
+Which approach to use and how to configure it depends on the requirements of the application
+as well as the features and limitations of the underlying cloud deployment platform.
+
+### Single control plane topology
```

Comment:
You need to add the biggest caveat here that its the users responsibility to setup DNS resolution for services in remote clusters (as we dont have service entries anymore). While the multicluster one comes with DNS resolution for services